pkg/cmd/roachprod-microbench/google/service.go

```diff
@@ -212,7 +212,7 @@ func (srv *Service) createRawSheet(
 			vals = append(vals, numCell(entry.Summaries[run].Center))
 		}
 		delta := comparison.FormattedDelta
-		if delta == "~" {
+		if delta == "~" || delta == "?" {
```

Comment:
It seems that `?` is not even documented in the public API -- looking at the code, we see that it's returned when `old == 0`. Do we know when that would be the case?